portal-web/test/functional/com/liferay/portalweb/macros/ABTest.macro

```diff
@@ -313,6 +313,17 @@ definition {
 			key_text = "Publish");
 	}
 
+	macro deleteABTestACSide {
+		Button.clickDelete();
+		
+		AssertTextEquals.assertPartialText(
+			locator1 = "Modal#MODAL_CONTENT",
+			value1 = "Are you sure you want to delete this test? You will permanently lose all test data and results collected from this test. You will not be able to run this test. You will not be able to undo this operation.");
```

Comment:
Hey @manuferreira, your macro is an action macro, I don't think it's good to put an `Assert` inside an action macro. You can put this `Assert` directly in the test or ask the frontend team to create a unit test to validate it if they don't have it. What do you think?

Reply:
https://github.com/liferay-ac/liferay-portal/pull/729/commits/d6645aea29e2afe9f54f666a782ff4fae708c3e8